source/renderer/app/stores/NewsFeedStore.js

```diff
@@ -172,11 +172,11 @@ export default class NewsFeedStore extends Store {
     this.fetchingNewsFailed = fetchingNewsFailed;
   };
 
-  @action proceedNewsAction = (newsItem, e: MouseEvent) => {
+  @action proceedNewsAction = (newsItem: NewsItem, e: MouseEvent) => {
+    const { currentLocale } = this.stores.profile;
     const { url, route, event } = newsItem.action;
-
     if (url) {
-      this.stores.app.openExternalLink(url, e);
+      this.stores.app.openExternalLink(url[currentLocale], e);
```

Comment:
@tomothespian It was also throwing flow test error and if you debug it, you would know why it was changed.

Reply:
@yakovkaravelov that is weird because that is handled in the store. I'm afraid that open external link handler will be broken with this approach.  Can you investigate that? 
We can't add some additional code just to make flow to be happy. We need to find the right reason for the flow error

Reply:
@tomothespian Just checked this and confirmed my approach is correct.
`url` is `NewsTranslations` object.
https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/api/news/types.js#L3
So `url[currentLocale]` should be used.

Reply:
@yakovkaravelov please make sure to test this manually. Reading type declaration is one thing, but checking the actual data is a different one.